kuka_kr10_support/urdf/kr10r1420_macro.xacro

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find kuka_resources)/urdf/common_constants.xacro"/>
+  <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
+
+  <xacro:macro name="kuka_kr10r1420" params="prefix">
```

Comment:
```suggestion
  
  <xacro:macro name="kuka_kr10r1420" params="prefix">
    <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
```
this can be included inside the macro

Reply:
restructured it, also in KR10R1100 SIXX.